public/css/theme-arc-green.css

```diff
@@ -42,7 +42,7 @@ a:hover{color:#a0cc75}
 .ui.attached.table{border:1px solid #304251;background:#304251}
 .feeds .list ul li:not(:last-child){border-bottom:1px solid #333640}
 .feeds .list ul li.private{background:#353945;border:1px solid #333640}
-.ui.secondary.menu .dropdown.item:hover,.ui.secondary.menu .link.item:hover,.ui.secondary.menu a.item:hover{color:#fff}
+.ui.secondary.menu .dropdown.item:hover,.ui.secondary.menu .link.item:hover,.ui.secondary.menu .active.item:hover,.ui.secondary.menu a.item:hover{color:#fff}
```

Comment:
This file is generated from [/public/less/themes/arc-green.less](https://github.com/go-gitea/gitea/blob/master/public/less/themes/arc-green.less). Please edit the less file, call `make css` and commit changes from all files.

Reply:
> This file is generated from [/public/less/themes/arc-green.less](https://github.com/go-gitea/gitea/blob/master/public/less/themes/arc-green.less). Please edit the less file, call `make css` and commit changes from all files.

Right, done.